spec/configuration_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'spec_helper'
+
+describe ActionTracker::Configuration do
+  describe '#track_events flag' do
+    it 'default value is true' do
+      expect(ActionTracker::Configuration.new.track_events).to eq(true)
```

Comment:
I think you should write the tests using the `ActionTracker.configure` method, passing a block like you would do in a real application. This way you are testing all the configuration code, not only `ActionTracker::Configuration` class. What do you think?